src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts

```diff
@@ -86,14 +86,10 @@ export class DotPageStateService {
      * @returns Observable<DotRenderedPageState>
      * @memberof DotPageStateService
      */
-    get(url: string, languageId?: number): Observable<DotRenderedPageState> {
-        const options: DotRenderPageOptions = {
-            url: url
-        };
-
-        if (languageId) {
-            options.viewAs = {
-                language_id: languageId
+    get(options: DotRenderPageOptions | string): Observable<DotRenderedPageState> {
```

Comment:
We talked about this... `DotRenderPageOptions | string` doesn't make sense here.